app/controllers/userController.js

```diff
@@ -1,13 +1,15 @@
+import httpStatus from 'http-status'
 import UserService from '../services/userService'
+import responseWrapper from '../../lib/helpers/response-wrapper'
 
 class UserController {
   static async signup(req, res, next) {
-    try {
-      await UserService.createUser(req.body)
-      return res.status(201).json({ message: 'signup successful' })
-    } catch (error) {
-      return next(error)
-    }
+    await UserService.createUser(req.body)
+      .then(user => {
+        return { data: user }
```

Comment:
here data will be `null` as sequelize create method of the model does not return anything unless you explicitly find the user you created and return them. [read more](https://sequelize.org/master/manual/models-usage.html#-code-findorcreate--code----search-for-a-specific-element-or-create-it-if-not-available)